Description: Fix FTBFS with gcc-4.7
 Fix FTBFS with gcc 4.7 by fixing missing <unistd.h> include.
Origin: vendor
Bug-Debian: http://bugs.debian.org/667360
Forwarded: no
Author: Cyril Brulebois <kibi@debian.org> 
Last-Update: 2012-05-15

--- a/rsakeyfind.cpp
+++ b/rsakeyfind.cpp
@@ -10,6 +10,7 @@
 #include <sys/mman.h>
 #include <string.h>
 #include <iostream>
+#include <unistd.h>
 
 #ifdef __FreeBSD__
 #include <err.h>
